### ISSUE TITLE
I can't set the password on rasa x

### ISSUE THREAD

**Issue by harunkelesoglu** [RasaHQ/rasa#3657]
**Rasa version**:
Rasa 1.0.0
**Python version**: 
3.6.8
**Operating system** (windows, osx, ...):
Linux Ubuntu 
**Issue**:
I'm trying install the rasa x community according to this http://rasa.com/docs/rasa-x/deploy/#quick-installation

I cant set password on 4.step. When i run the following command 
```
$ cd /etc/rasa
$ sudo python rasa_x_commands.py set_password PASSWORD
```
got this error :
```
usage: rasa_x_commands.py [-h] {create,delete} ...
rasa_x_commands.py: error: invalid choice: 'set_password' (choose from u'create', u'delete')
``
`

**Comment by akelad** [comment]
Should also be fixed in the latest release (1.0.3) -- @wochinge please confirm

**Comment by wochinge** [comment]
Related to https://github.com/RasaHQ/rasa/issues/3577 but is actually an error in the docs. Thanks for catching this. Will be fixed in the next release.
